.github/workflows/ci.yml

```diff
@@ -286,6 +286,10 @@ jobs:
         run: py -3 test\util\rpcauth-test.py
 
       - name: Run functional tests
-        env:
-          TEST_RUNNER_EXTRA: ${{ github.event_name != 'pull_request' && '--extended' || '' }}
-        run: py -3 test\functional\test_runner.py --jobs $env:NUMBER_OF_PROCESSORS --ci --quiet --tmpdirprefix=$env:RUNNER_TEMP --combinedlogslen=99999999 --timeout-factor=$env:TEST_RUNNER_TIMEOUT_FACTOR $env:TEST_RUNNER_EXTRA
+        # Only run functional tests in master.
```

Comment:
I reckon this is not correct. The job will run on every push to our version branches and in personal repositories for any branch as well.

Maybe "Do not run functional tests for pull requests." or similar?

Reply:
Ah, ok, @MarcoFalke actually suggested to do what the comment says. So I instead adapted the if check to really only run in master (I hope). Does that work for you as well or should I rather change the comment?

Reply:
> So I instead adapted the if check to really only run in master (I hope).

Why skip version branches?

Also I found useful my personal repo CI in my everyday work.